packages/framework/ai-collab/src/README.md

```diff
@@ -0,0 +1,188 @@
+## Description
```

Comment:
Is there a reason these docs can't go in the package-level README? We generally don't have `src`-level READMEs, except to document specific directories.

Reply:
There isn't. Moved the README to the root level in commit a79e33c651f4084ad3e7f7c04689ef61d95f4cd7

Reply:
I'm still seeing this document in the most recent commit.

Reply:
Huh, I updated the package level one and forgot to delete the /src one. it should be updated now in commit 3177336e7b111b192b05835c8af5d2c3e177f952